spec/rubocop/cop/rspec/void_expect_spec.rb

```diff
@@ -44,4 +44,18 @@
       end
     RUBY
   end
+
+  it 'ignores unrelated expect method' do
+    expect_no_offenses(<<~RUBY)
+      MyObject.expect
+    RUBY
+  end
+
+  describe '#void?' do
```

Comment:
Is there another way to test this private method? :x:

Reply:
That was my question for you!

Reply:
This is the guard clause in the private method that is not being invoked:

<img width="541" alt="Screenshot 2024-10-14 at 6 49 05 AM" src="https://github.com/user-attachments/assets/63edd01a-6d0e-490b-b3eb-ec97a605454b">

Reply:
So no spec fails if we remove this line from the cop? 😕

Reply:
No spec fails, but I don't think its advisable to remove it. 

I plastered this with tests and got the coverage. 

Also some of the tests seem to indicate behavior that might not be wanted (outside of it blocks, etc). But in this PR I see my role as documenting the existing implementation to unblock easier test coverage. 

Take a look and see if there's anything you'd want to amend in a subsequent PR.

Reply:
I fixed the clarity here by updating the Context name to include the Prefixes for this spec